format.go

```diff
@@ -16,7 +13,7 @@ const (
 type Units uint
 
 const (
-	UnitNone Units = iota
```

Comment:
It forces you to use the exported API. I personally like it. Also helps making sure you test what should be tested (i.e I should not be testing `formatBytes` function).

Another example: duplicating the constant definitions for `KiB`, `MiB`, etc... inside the test package. If the main package changes `bytesInMiB` to a wrong value, tests will fail. If the test relied on the internals, test would pass.

I am happy putting them in the main package however. You tell me.